yarn.lock

```diff
@@ -1302,6 +1302,7 @@
   dependencies:
     "@chakra-ui/utils" "1.10.4"
 
+<<<<<<< HEAD
```

Comment:
this shouldn't be here. [try this](https://stackoverflow.com/a/42939114/3624290) to fix it